crates/telio-traversal/src/endpoint_providers/stun.rs

```diff
@@ -125,10 +126,7 @@ impl<Wg: WireGuard, E: Backoff> StunEndpointProvider<Wg, E> {
 
             s.exponential_backoff.reset();
 
-            // Start STUN session, but only if we have at least one STUN server configured
-            if !s.servers.is_empty() && s.start_stun_session().await.is_err() {
-                telio_log_error!("STUN session could not be started");
-            }
+            s.transition_to_wait_for_wg().await;
```

Comment:
Why is checking if there's at least one STUN server configured no longer necessary?

Reply:
Good point - I guess it should be moved a bit though